docs/concepts/credentials/2_claiming.md

```diff
@@ -0,0 +1,83 @@
+---
+id: claiming
+title: Claiming
+---
+There are three actors in the KILT workflow: Claimers, Attesters and Verifiers.
+
+As KILT is an open system: any entity can make a claim about anything. Therefore, as in the real world, a claim only has value if the verifier trusts the attester.
+
+In KILT, claims are based on claim types (CTypes). A claimer can either create a new CType or, when KILT is established and standard CTypes are available, may use an existing CType when creating their claim.
+
+## CTypes
```

Comment:
Why are we re-presenting CTypes again? Wasn't the previous section enough?

Reply:
I guess it is just here, in case the reader didn't read the previous section, But I can also remove it and just reference the ctype section, where needed.